scripts/shared/lib/deploy_bundle

```diff
@@ -0,0 +1,249 @@
+  components+=$([[ ${globalnet} == "true" ]] && echo ", globalnet" || echo "")
+
+  ### Create the Broker instance
+cat <<EOF | kubectl apply -f -
+apiVersion: submariner.io/v1alpha1
+kind: Broker
+metadata:
+  name: submariner-broker
+  namespace: ${SUBM_NS}
+spec:
+  components: [${components}]
+  defaultGlobalnetClusterSize: 8192
+  globalnetEnabled: ${globalnet}
+  globalnetCIDRRange: 169.254.0.0/16
+EOF
+
+  echo "[INFO] Wait for the broker readiness..."
+  if ! (timeout 5m bash -c "until kubectl --context=${cluster} get brokers.submariner.io submariner-broker -n ${SUBM_NS} > /dev/null 2>&1; do sleep 10; done"); then
+      echo "[ERROR] Broker is not ready."
+      exit 1
+  fi
+
+  brokerClientSecret=$(kubectl -n ${BROKER_NAMESPACE} get secrets -o json | jq -r -c '[.items[] | select(.metadata.annotations."kubernetes.io/service-account.name"=="'${BROKER_CLIENT_SA}'") | select(.data.token != null) | select(.data."ca.crt" != null)] | .[0]')
+  BROKER_K8S_API_SERVER=$(kubectl get endpoints kubernetes -n default -o jsonpath="{.subsets[0].addresses[0].ip}:{.subsets[0].ports[?(@.name=='https')].port}")
+  BROKER_K8S_API_SERVER_TOKEN=$(echo "${brokerClientSecret}" | jq -r '.data.token' | base64 --decode)
+  BROKER_K8S_CA=$(echo "${brokerClientSecret}" | jq -r '.data."ca.crt"')
+
+}
+
+function install_subm() {
+  if [[ ${cluster_subm[$cluster]} != "true" ]]; then
+        echo "Skipping installation as requested in cluster settings"
+        return
+  fi
+
+  if kubectl wait --for=condition=Ready pods -l app=submariner-operator -n "${SUBM_NS}" --timeout=60s > /dev/null 2>&1; then
+        echo "[WARN](${cluster}) Submariner already installed, skipping installation..."
+        return
+  fi
+
+  if ! (timeout 5m bash -c "until kubectl --context=${cluster} get crds submariners.submariner.io > /dev/null 2>&1; do sleep 10; done"); then
+      echo "[ERROR] Submariner CRD was not found."
+      exit 1
+  fi
+
+  # Create the Submariner instance
+  echo "[INFO](${cluster}) Deploy Submariner"
+cat <<EOF | kubectl apply -f -
+apiVersion: submariner.io/v1alpha1
+kind: Submariner
+metadata:
+  name: submariner
+  namespace: ${SUBM_NS}
+spec:
+  serviceCIDR: "${service_CIDRs[$cluster]}"
+  clusterCIDR: "${cluster_CIDRs[$cluster]}"
+  globalCIDR: "${global_CIDRs[$cluster]}"
+  clusterID: "${cluster}"
+  debug: true
+  natEnabled: true
+  serviceDiscoveryEnabled: ${service_discovery}
+  broker: "k8s"
+  brokerK8sApiServer: "${BROKER_K8S_API_SERVER}"
+  brokerK8sApiServerToken: "${BROKER_K8S_API_SERVER_TOKEN}"
+  brokerK8sRemoteNamespace: "${BROKER_NAMESPACE}"
+  brokerK8sCA: "${BROKER_K8S_CA}"
+  cableDriver: "libreswan"
+  ceIPSecPSK: "${IPSEC_PSK}"
+  ceIPSecDebug: true
+  ceIPSecIKEPort: ${CE_IPSEC_IKEPORT}
+  ceIPSecNATTPort: ${CE_IPSEC_NATTPORT}
+  namespace: "${SUBM_NS}"
+  repository: "${SUBM_IMAGE_REPO}"
+  version: "${SUBM_IMAGE_TAG}"
+  connectionHealthCheck:
+    enabled: true
+    intervalSeconds: 1
+    maxPacketLossCount: 5
+EOF
```

Comment:
This would be better if it was in a real YAML file which gets validated. You can use `render_template` function from `clusters.sh`